.gitignore

```diff
@@ -12,6 +12,7 @@
 
 # microsoft c/c++ intellisense (disabled in favor of clangd)
 /.vscode/c_cpp_properties.json
+/.vscode/settings.json
```

Comment:
Is settings.json still being autogenerated? What is its contents? Haven't had this happen to me before

Reply:
Yeah I just did a fresh rebuild of everything and it showed up for me. I can check the contents when I get home, I didn't look too closely at what was in there

Reply:
it doesn't for me, maybe due to a difference in our personal settings

Reply:
That's probably what it is, do you think settings.json should be tracked?

Reply:
maybe just ignore the entire directory since we don't use it. All settings should be in the code-workspace file

Reply:
Yeah thats a good point, I'll do that